### DISCUSSION TITLE
Welcome to listd Discussions!

### DISCUSSION THREAD

**Discussion by w3cj** [CodingGarden/listd#17] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by puzzle-it-nu** [comment]
Hi CJ,

I'm (originng) a beginner JS/TS programmer, mostly working as a systems admin. I like your streams and I'm implementing Listd as a Nuxt3 app instead of a Sveltekit one.
So currently using Nuxt3+Vuetify3+i18n-vue(nuxt)+sidebase-nuxt-auth
